editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -749,7 +765,11 @@ export const GridControls = controlForStrategyMemoized<GridControlsProps>(({ tar
           }
 
           return (
-            <div key={`grid-${EP.toString(grid.elementPath)}`} style={style}>
+            <div
+              key={`grid-${EP.toString(grid.elementPath)}`}
+              id={`grid-${EP.toString(grid.elementPath)}`}
```

Comment:
This isn't necessary if we use the metadata to look up grids

Reply:
same here - we need to query the grid placeholder (the one that displays the placeholder cells, since we need the cells and not the elements), so we can't use the metadata for it

Reply:
done in https://github.com/concrete-utopia/utopia/pull/6160/commits/dd7e663d8985a530ad6b33d7f481fdb5a4fd2563